raml-definition/spec-1.0/datamodel.ts

```diff
@@ -13,7 +13,7 @@ export enum LocationKind{
 }
 
 export class ExampleSpec extends Common.RAMLLanguageElement {
```

Comment:
Why does it still have to extend from `RAMLLanguageElement`? It seems that the class calls out all necessary properties explicitly (displayName, description, annotation).